doc/classes/Viewport.xml

```diff
@@ -1,11 +1,11 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="Viewport" inherits="Node" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Base class for viewports.
+		Abstract base class for viewports. Encapsulates drawing and interaction with a game world.
```

Comment:
```suggestion
		Abstract base class for viewports. Encapsulates drawing and interactions with a game world.
```
probably

Reply:
I think it makes more sense to say it encapsulates "interaction" itself, not the individual interactions.